src/applications/ezr/config/chapters/veteranInformation/contactInformation.js

```diff
@@ -27,7 +27,12 @@ export default {
           pattern: content['phone-number-error-message'],
         },
       },
-      email: emailUI(),
+      email: emailUI({
+        errorMessages: {
+          pattern:
+            'Enter a valid email address without spaces using this format: email@domain.com',
```

Comment:
Can you move this to the content.json file? We don't want magic strings in these files.